contentcuration/contentcuration/models.py

```diff
@@ -458,7 +462,7 @@ def generate_storage_url(filename, request=None, *args):
     # and let nginx handle proper proxying.
     if run_mode == "k8s":
         url = "/content/{path}".format(
-            bucket=settings.AWS_S3_BUCKET_NAME,
+            # bucket=settings.AWS_S3_BUCKET_NAME,
```

Comment:
@jayoshih Just noticed this line in a commit for adding email error handling. Could this have been by mistake?

Reply:
It's an unused argument in the string formatting, so I imagine a linter was complaining?